development.rst

```diff
@@ -12,17 +12,86 @@ Development of the specification
 is intended to be an open process.
 Contributions are welcome from anyone.
 Problems or feature requests
-should be filed against the github project.
+should be filed against the GitHub project.
 
-We do not have a mailing list at this time,
-although we expect to start one
-once there is significant community involvement.
-For now, please direct comments to `the editor`_.
+Much activity and discussion takes place on GitHub,
+and we encourage use of Issues and Discussions
+as a primary communication channel.
+Some members of the community are active
+in the #ecosystem_evolution channel
+of `CppLang's Slack <https://cpplang.slack.com>`_.
+We can also be reached via our `mailing list`_.
+
+In addition, the ecosystem evolution group
+typically has at least one monthly video-conference call.
+While not strictly limited to CPS discussion,
+matters regarding the specification are sometimes discussed there.
+Additional, more CPS-focused meetings may also occur.
+See the `mailing list`_ for details.
+
+Contributing
+''''''''''''
+
+Pull requests are welcome,
+but please be aware that changes
+which have not been previously discussed
+may not be desirable, and may be rejected.
+At a minimum, if you submit a pull request
+that has not been previously discussed,
+it is likely that you will be asked to make changes
+before your proposal is accepted.
+
+When editing the documentation,
+please keep in mind that whitespace,
+and line breaks in particular,
+are not meaningful to the generated HTML.
+Accordingly, it is preferred to use these
+in a way that aims to reduce diff churn
+if and when the text is subsequently revised.
+Therefore, do not try to use all available space
+or slavishly wrap at a particular column.
+Rather, keep lines shorter
+and try to break at "natural" boundaries,
+and **always** break after a sentence.
+As a semi-hard limit, reST text
+should avoid exceeding 79 characters on a line;
+however, "properly" broken lines
+should rarely approach that limit.
+
+Commits
+^^^^^^^
+
+Commit messages shall consist of a one-line summary,
+a blank line, and a longer description.
+Summaries should ideally be 50 characters or less,
```

Comment:
Same comment. Should we add a linter or something to enforce these standards? It would be hard for me to consistently enforce that standard reviewing PRs

Reply:
This would be more plausible to machine-enforce. Kitware has a tool that runs on GitLab. There is [some work](https://github.com/mathstuf/ghostflow-action-sandbox) getting it to also work with GitHub, but the current functional state is not clear to me. Someone else has probably also looked into this, but I wouldn't know where to find such work.

In any case, the tool should either be advisory-only, or it needs to be possible to override it, since there can be exceptions. The 50-character summary limit in particular is more of a "strong recommendation" than a hard limit. The 72-character limit however should only be violated in cases of more than 72 characters of text into which insertion of a line break *anywhere* is contraindicated.